### ISSUE TITLE
Page title missing

### ISSUE THREAD

**Issue by developpeurweb** [SAP/spartacus#15917]
**Describe the bug**

- The page title for the page on the screenshot vocalizes the URL (it has no name) and it bloats the message when successfully adding to the cart, and the alert message is heard only **after** the whole URL narration.

**Tell us the version of Spartacus**
- Library version: 5.0-develop

**Expected behaviour**

- Page has title

**Screenshots**

![Screen Shot 2022-06-16 at 6 27 56 PM](https://user-images.githubusercontent.com/15113219/176270118-bc1b6ca9-63ba-4771-b75f-c233d256e303.png)

**Comment by Changsuwan** [comment]
Pull Request: https://github.tools.sap/cx-commerce/spartacussampledata/pull/230
